### DISCUSSION TITLE
Try to compile..and get errors

### DISCUSSION THREAD

**Discussion by chanh2018** [LawnchairLauncher/lawnchair#3923] [General]
Why I have this error when open the code..

import app.cash.licensee.LicenseeTask
import com.android.build.gradle.api.AndroidBasePlugin
import org.jetbrains.kotlin.gradle.tasks.KotlinCompile

Cause: error=2, No such file or directory

**Comment by SuperDragonXD** [comment]
Hello. I know that this is a bit late, but this discussion is a bit stale. As such, will close it. If you are still attempting to build Lawnchair, please create a new discussion with screenshots of what you are trying to do and the exact error codes you have.
